lib/localeapp/exception_handler.rb

```diff
@@ -1,16 +1,18 @@
 module Localeapp
   class ExceptionHandler
     def self.call(exception, locale, key, options)
+      keys = Array(key)
+      keys = keys.map { |key| ERB::Util.html_escape(key.to_s) }
```

Comment:
Those 2 lines can be merged, imho: `keys = Array(key).map { |key| ERB::Util.html_escape(key.to_s) }`